_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,107 @@
+the news.
+
+## Why Encrypt Data in OpenSearch?
+
+Search services are often an underprotected backdoor way to gain access to sensitive data. They frequently hold
+personal information like names and addresses. They can hold log data that includes web form submissions. They
+may hold communications records like emails, customer messages, and chats. They may also hold insider-trading
+fodder like roadmap info, financial projections, legal documents, and much more.
+
+A search service is akin to an attractive nuisance. Just as a swimming pool without a fence will attract
+neighborhood kids, search data without encryption will attract hackers and employees.
+
+The employee side is what we call the “curious insider” problem where trusted users ignore policy and bypass or
+abuse application-level access controls to peek at data they shouldn't. Database administrators, ops, customer
+support, engineering – in most companies, many of these roles can sneak peeks at sensitive data. And these
+trusted folks who scratch the itch of curiosity can get a company in just as much trouble as a hacker.
+
+![Cloaked Search Proxy]({{ site.baseurl }}/assets/media/blog-images/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search/cloaked-search-proxy.png){:class="img-centered"}
+
+Access to sensitive data should always be controlled. Backdoor access should be eliminated whenever possible.
+That includes controlling access by admins. With application-layer encryption, a company can build in these
+controls for all users and administrators, and can fortify legitimate access with unbypassable audit trails.
+
+## Protecting Search Data with Cloaked Search
+
+At IronCore Labs, we provide tools and technologies that help companies protect the data they hold with
+application-layer encryption while keeping that data usable. One way we do that is to enable search over
+encrypted data.
+
+[Cloaked Search](https://ironcorelabs.com/products/cloaked-search/) by IronCore Labs is a drop-in proxy that
+sits in front of OpenSearch and automatically encrypts the sensitive data before it gets to the OpenSearch
+service. It's application-layer encryption for search. To search the data then requires a valid key. If you
+don't have a valid key, the data in OpenSearch is useless.
+
+![Cloaked Search Benefits]({{ site.baseurl }}/assets/media/blog-images/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search/benefits.png){:class="img-entered"}
+
+## Features and Functionality
+
+Cloaked Search is easy to deploy, simple to configure, **and immediately adds protection to the private data
+ held in your search service**. It comes packaged as a Docker container which deploys in your infrastructure.
+Your application connects to it and it then connects to the OpenSearch service. 
+
+No plugins or other modifications are needed to OpenSearch, so it works as well with Amazon's service as with
```

Comment:
It looks like there may be a missing word(s) in the first clause prior to "OpenSearch".